packages/react/src/QuestionnaireForm/QuestionnaireFormItem/QuestionnaireFormItem.tsx

```diff
@@ -37,6 +37,7 @@ import { QuestionnaireFormContext } from '../QuestionnaireForm.context';
 export interface QuestionnaireFormItemProps {
   readonly item: QuestionnaireItem;
   readonly index: number;
+  readonly isRequired?: boolean;
```

Comment:
Can we change this to just `required` ?

We try to reserve the `is` prefix for getter functions.  Boolean variables should just be the word itself.